<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostureEditorTab</class>
 <widget class="QWidget" name="PostureEditorTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>3</number>
     </property>
     <widget class="QWidget" name="glwidgetHolderLayout" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="sideWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="PostureSideWidget" name="managePostures" native="true"/>
       </item>
       <item>
        <widget class="PostureJointSideWidget" name="editPosture" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PostureSideWidget</class>
   <extends>QWidget</extends>
   <header location="global">GUI/SideWidgets/PostureSideWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PostureJointSideWidget</class>
   <extends>QWidget</extends>
   <header>GUI/SideWidgets/PostureJointSideWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
